### DISCUSSION TITLE
Invoke-PnPSiteTemplate - the field was found invalid

### DISCUSSION THREAD

**Discussion by alexc-MSFT** [pnp/powershell#1852] [Q&A]
### Reporting an Issue or Missing Feature
Reporting an Issue

### Expected behavior 
I am trying to apply a site template using Invoke-PnPSiteTemplate, I expect the template to be applied.  

### Actual behavior 
This cmdlet fails with a 'the field was found invalid' error. I am experiencing this issue on two of my customers machines yet on my machine it works completely fine using the same template. I have inspected the template in detail and cannot find any field that would cause this issue.

I fully appreciate that I should also raise this in the pnpframework repo however that repo has tons of issues which are unresolved and have been left stagnant so I am worried this will never be fixed. 

`WARNING: ListInstance Style Library is a Style Library of a site with NoScript enabled and will be skipped
Failed to configure the SharePoint site {0} The field was found invalid:
At C:\Users\adm_e606495\Desktop\New folder\Source\Scripts\deploy.ps1:671 char:9
+ throw('Failed to configure the SharePoint site {0}', $_.Excep ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo : OperationStopped: (System.Object[]:Object[]) [], RuntimeException
+ FullyQualifiedErrorId : Failed to configure the SharePoint site {0} The field was found invalid:`

Worth noting this cmdlet is being run from inside my own script, the same error happens outside of it. 

### Steps to reproduce behavior
Invoke-PnPSiteTemplate -Path "C:\template.xml" -ClearNavigation

### What is the version of the Cmdlet module you are running?
1.10.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
@alexc-MSFT 

> I am experiencing this issue on two of my customers machines yet on my machine it works completely fine using the same template.

Then we can agree that the cmdlet works and there's something on your customers' environments that is giving you issue? 

Not sure what we can do here. So I'll transfer this as a discussion.

**Comment by alexc-MSFT** [comment]
@veronicageek potentially yes but getting the bottom of what it is about their environment is the biggest problem.

Would appreciate any ideas, they are running PS 5.1 and afaik Windows 10, we can of course obtain the exact version. Other than that I'm not sure what else could be different.

Is there any way to further dig into what happens behind the scenes? Would allow me to perhaps see what field it has an issue with? 

Cheers

**Reply by AndersRask** [reply]
Hi @alexc-MSFT try and enable trace when running the provisioning

Set-PnPTraceLog -On -LogFile traceoutput.txt

Note there are more options regarding level of output etc.
Use -Off to turn it off.

You can also write to console, but I find the log file easier to work with

hth
Anders Rask

**Reply by alexc-MSFT** [reply]
Amazing thanks Anders - I had no idea you could do that!
